src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -261,7 +263,7 @@ void CascadiaSettings::_ValidateDefaultProfileExists()
 
         // _temporarily_ set the default profile to the first profile. Because
         // we're adding a warning, this settings change won't be re-serialized.
-        GlobalSettings().DefaultProfile(_profiles[0].GetGuid());
+        GlobalSettings().DefaultProfile(GUID(_profiles[0].Guid()));

```

Comment:
```suggestion
        GlobalSettings().DefaultProfile(GUID{ _profiles[0].Guid() });
```

don't use `()` for constructor calls

Reply:
```
1>D:\Terminal\src\cascadia\TerminalApp\CascadiaSettings.cpp(266,67): error C2440: '<function-style-cast>': cannot convert from 'initializer list' to 'GUID'
1>D:\Terminal\src\cascadia\TerminalApp\CascadiaSettings.cpp(266,45): message : Invalid aggregate initialization
```
GlobalAppSettings is already the next PR that takes in a winrt::guid. Then this problem doesn't exist. Can we just hold off on this one?